docs/general/networking/caddy.md

```diff
@@ -13,6 +13,30 @@ You can reverse proxy to Jellyfin either with or without a config file, and eith
 
 If you aren't familiar with Caddy yet, check out its [Getting Started](https://caddyserver.com/docs/getting-started) guide.
 
+:::caution
+There are a some guides that have a Caddyfile which includes the `tls` section with a DNS Provider API Key, as shown the following example.
+
+```Caddyfile
+example.com {
+    reverse_proxy 127.0.0.1:8096
+    // highlight-start
+    tls {
+        dns <DNS Provider> <API Token>
+    }
+    // highlight-end
+}
+```
+
+Please proceed with caution when using this option:
+
+- This will **NOT** automatically update your DNS records if you have a dynamic IP.
+- This is **only** necessary if you **cannot** open ports 80 and 443
+- Misconfiguration can lead to **compromised domains and/or accounts**.
```

Comment:
Is there a source or small explanation that you can back this up with? In what way can it lead to a compromise for example. Ignore this comment if this might be too much.

Reply:
I think it would be too long as this section is intended to be a short warning

Reply:
And also I think it should be very obvious simply by looking at what permissions are available and can be enabled for the API keys.